src/python/pants/jvm/target_types.py

```diff
@@ -309,6 +310,17 @@ class JunitTestSourceField(SingleSourceField, metaclass=ABCMeta):
 # -----------------------------------------------------------------------------------------------
 
 
+class ReproducibleJar(BoolField):
+    alias = "reproducible"
+    default = False
+    help = softwrap(
+        """
+        Indicate if the deploy jar should be reproducible by stripping timestamps
+        and metadata from the deploy jar.
```

Comment:
```suggestion
        Indicate if the deploy jar should be reproducible by stripping timestamps
        and metadata from the deploy jar.
```
```suggestion
        Indicate that the deploy jar should be made reproducible by stripping timestamps
        and metadata.
```

Reply:
Maybe mention the name of the tool that strips the metadata? (Which act as a cross-reference for users reading this message to the option subsystem for this tool.)